elasticsearch_dsl/__init__.py

```diff
@@ -99,6 +100,8 @@
 __versionstr__ = ".".join(map(str, VERSION))
 __all__ = [
     "A",
+    "Agg",
```

Comment:
There are some classes that were not directly exported at the package level because the user rarely needs to directly create them. Now that we have to write type hints these are sometimes needed, so I decided to add them so that people don't go out to fish them out from their internal locations.